docs/site/BelongsTo-relation.md

```diff
@@ -75,6 +75,14 @@ export interface OrderRelations {
 export type OrderWithRelations = Order & OrderRelations;
 ```
 
+
+Definition of `belongsTo` relation can also include explicit `keyFrom`, `keyTo` and `name` attributes.
+
+- keyFrom is foreign key of the referenced model, whereas keyTo is primary key of referenced model, name is the referenced model name. Usage of keyFrom and keyTo in a belongsTo relation is similar to the use in legacy datasource juggler.
+- In the above example for the model Order, keyFrom is `customerId` and keyTo is `id`. keyFrom defaults to `{decorated-belongsTo-property-name}`, keyTo defaults to `id` and the referenced model name defaults to the decorated property name after stripping the trailing `Id` suffix.
+- In case in the above example, the property name `customerId` has to be customized to say `cust_id`, then the keyFrom,  keyTo and name values needs to be explicitly defined as `@belongsTo(() => Customer, { keyFrom: 'customerId', keyTo: 'id'}, { name: 'customer' })`.
+
```

Comment:
It's the same for `Has Many` relation, 

By default, the `keyFrom` field for `HasMany` relation would be the id property in the source model. In this case, it would be the id property `id` in model `Customer`. 
By default, the `keyTo` field for `HasMany` relation would be the foreign key in the target model. In this case, it would be the `customerId` in model `Customer`.

If we have a different foreign key name  other than `customerId`, we should specify it via `@hasMany` decorator:
```ts
class Customer...{
...
@hasMany(()=> Order, {keyTo: 'my_custom_id'})
orders: Order[];
}
```
We need to specify `keyTo` here to overwrite the default value. ( we can also specify the name of the relation if needed)

But don't worry about the Has Many relation in this PR. I just think it's helpful to understand `keyFrom` and `keyTo` together with Has Many :smile:

Reply:
keyFrom has become a dummy parameter. I can add the implications on hasMany if required.

Reply:
I am afraid that this is addressed. I believe that `keyFrom` works.